### PR TITLE
/api/events endpoint can be used to receive events not only from client

### PR DESCRIPTION
**Pull request by Eldies**
### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have created a changelog fragment 
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

## Summary by CodeRabbit

- **New Features**
	- Introduced a new permission scope for sending external events.
	- Added a method to log external events, accessible via a POST request.

- **Bug Fixes**
	- Ensured existing event logging functionalities remain intact while integrating new features.

### CODE REVIEW

File: cvat/apps/events/views.py [hunk 1]
```diff
@@ -44,6 +48,26 @@ def create(self, request):
 
         return Response(serializer.validated_data, status=status.HTTP_201_CREATED)
 
+    @extend_schema(summary='Log external events',
```
Comment (nmanovic) [inline cvat/apps/events/views.py:51]: Do we really need several end-points to send events (add a row into Clickhouse)?

File: cvat/apps/events/permissions.py [hunk 2]
```diff
@@ -10,10 +10,12 @@
 from cvat.apps.iam.permissions import OpenPolicyAgentPermission, StrEnum
 from cvat.utils.http import make_requests_session
 
+
 class EventsPermission(OpenPolicyAgentPermission):
     class Scopes(StrEnum):
         SEND_EVENTS = 'send:events'
         DUMP_EVENTS = 'dump:events'
+        SEND_EXTERNAL_EVENTS = 'send-external:events'
```
Comment (azhavoro) [inline cvat/apps/events/permissions.py:18]: Please update the rego rules to reflect this change

**Comment by azhavoro** [comment]
Consider adding some tests for this endpoint

File: cvat/apps/events/permissions.py (hunk 2 above)
Comment (Eldies) [reply cvat/apps/events/permissions.py:18]: I planned it to be only accessible by admins, and rego rules do not need to be changed for it.
But due to comments above I will try to use the existing endpoint somehow, and just drop this scope

File: cvat/apps/events/views.py (hunk 1 above)
Comment (Eldies) [reply cvat/apps/events/views.py]: reused existing endpoint

File: cvat/apps/events/permissions.py (hunk 2 above)
Comment (Eldies) [reply cvat/apps/events/permissions.py]: removed it

**Comment by azhavoro** [comment]
@Eldies LGTM, but please add a changelog entry

**Review by azhavoro — APPROVED** [review]

**Pull request merged**